pkgs/by-name/ze/zerotierone-desktop-ui/package.nix

```
@@ -0,0 +1,95 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, gnumake
+, gcc
+, zlib
+, freetype
+, fontconfig
+, gtk3
+, libappindicator
+}:
+
+
+let
+  pname = "zerotierone-desktop-ui";
+  version = "2024.9.0"; # Infrequent updates and does not seem to be released often
```

```suggestion
  version = "1.8.3-unstable-2024-09-26";
```
Please review the [Versioning Guidelines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning).

---

thanks for the reference in the guide, changed accordingly